src/System.Runtime.Numerics/tests/BigInteger/Comparison.cs

```diff
@@ -553,6 +554,11 @@ private static void VerifyComparison(BigInteger x, UInt64 y, int expectedResult)
                 Assert.Equal(x.GetHashCode(), ((BigInteger)y).GetHashCode());
                 Assert.Equal(x.ToString(), ((BigInteger)y).ToString());
             }
+            else
+            {
+                Assert.NotEqual(x.GetHashCode(), ((BigInteger)y).GetHashCode());
```

Comment:
Technically this isn't valid; it's possible (but unlikely) their hash codes could match even if their values aren't the same.  If we're depending on an implementation detail that guarantees they won't be the same, ok.